### PR TITLE
Ed gen wave

### PR DESCRIPTION
**Pull request by dudublad**

### CODE REVIEW

File: QT/sinewavedisplay.cpp [hunk 1]
```diff
@@ -39,12 +39,19 @@ SineWaveDisplay::SineWaveDisplay(QWidget *parent) : SoundDisplay(parent)
     frequencyLayout->addLayout(frequencyControlLayout);
     mainLayout->addLayout(frequencyLayout);
 
+    waveTypeSelector = new QComboBox(this);
```
Comment (dudublad) [inline QT/sinewavedisplay.cpp:42]: @Maxarreaga This is the stuff to change the wave type

File: backend/audiofile.cpp [hunk 2]
```diff
@@ -13,6 +13,7 @@ void AudioFile::openFile(QString filePath){
 
     this->input.setRate((double) (this->input.getFileRate() / stk::Stk::sampleRate()));
 
+    //TODO: FIX SAMPLING RATE FOR SOUND
```
Comment (dudublad) [inline backend/audiofile.cpp:16]: @luckuisha

File: QT/main.cpp [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
 int main(int argc, char *argv[])
 {
-    stk::Stk::setSampleRate(44100.0);
+    stk::Stk::setSampleRate(140000.0);
```
Comment (luckuisha) [inline QT/main.cpp]: 132300.0

File: backend/wave.h [hunk 4]
```diff
@@ -1,11 +1,3 @@
...
-    #define fSampling = 44100;
-#endif
-
-#ifndef tSampling
-    #define tSampling = 0.00002267573;
-#endif
-
 #ifndef M_PI
     #define M_PI 3.14159265358979323846
 #endif
```
Comment (luckuisha) [inline backend/wave.h]: use qt pi

**Review by luckuisha — CHANGES_REQUESTED** [review]

**Review by Maxarreaga — APPROVED** [review]
LGTM, we'll merge the front end this week

**Review by luckuisha — APPROVED** [review]

**Pull request merged**